examples/getting-started-movielens/03-Training-with-HugeCTR.ipynb

```diff
@@ -57,7 +57,7 @@
     "HugeCTR is a GPU-accelerated recommender framework designed to distribute training across multiple GPUs and nodes and estimate Click-Through Rates (CTRs).<br>\n",
```

Comment:
<p><code>We walked through the documentation, but it is useful to understand the API.</code></p><p> </p><p>This sentence is not correct/uptodate as we do not walk through the documentation anymore. We can just delete it and write something like.  </p><p> </p><p>We define our model in a model.py file (+add reason why we do it in a model.py file)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
done